package.json

```diff
@@ -1,27 +1,32 @@
 {
-  "name": "wc3maptranslator",
+  "name": "wc3maptranslator-ts",
   "version": "2.0.0",
```

Comment:
Increase the version number to `3.0.0` since this introduces breaking changes.

Reply:
-#4